examples_tests/39.DenoiserTonemapper/main.cpp

```diff
@@ -4,6 +4,7 @@
 #include <irrlicht.h>
 
 #include "CommandLineHandler.hpp"
+#include "../../include/irr/asset/filters/dithering/CPrecomputedDither.h"
```

Comment:
this should already be included in `irr/asset/asset.h"

Reply:
corrected